pkg/networkmanager/interfaces.js

```diff
@@ -1445,6 +1445,18 @@ PageNetworking.prototype = {
         $("#networking-add-bridge").click($.proxy(this, "add_bridge"));
         $("#networking-add-vlan").click($.proxy(this, "add_vlan"));
 
+        // HACK - hide "Add Team" if it doesn't work due to missing
+        //        bits
+        //
+        //        https://bugzilla.redhat.com/show_bug.cgi?id=1375967
+
+        $("#networking-add-team").hide();
+        // We need both the plugin and teamd
+        cockpit.script("(test -f /usr/lib64/NetworkManager/libnm-device-plugin-team.so || test -f /usr/lib/x86_64-linux-gnu/NetworkManager/libnm-device-plugin-team.so) && test -f /usr/bin/teamd", { err: "ignore" }).
```

Comment:
I suggest testing the shorter condition first and using line breaks:
```
        cockpit.script("test -f /usr/bin/teamd && " + 
                       "(test -f /usr/lib64/NetworkManager/libnm-device-plugin-team.so || test -f /usr/lib/x86_64-linux-gnu/NetworkManager/libnm-device-plugin-team.so)",
                       { err: "ignore" }).
```

or even
```
        var libFileOptions = [
            "test -f /usr/lib64/NetworkManager/libnm-device-plugin-team.so",
            "test -f /usr/lib/x86_64-linux-gnu/NetworkManager/libnm-device-plugin-team.so",
        ];
        cockpit.script("test -f /usr/bin/teamd && (" + libFileOptions.join(" || ") + ")",
                       { err: "ignore" }).
```